gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/functions/tick.mcfunction

```diff
@@ -0,0 +1,7 @@
+
+execute as @e[type=potion,nbt={Item:{tag:{gm4_lightning_in_a_bottle:1b}}}] at @s run function gm4_lightning_in_a_bottle:potion_tracker/spawn_tracker
+
+# kill lazy loaded markers
+execute as @e[type=marker,tag=gm4_liab_potion_tracker] run function gm4_lightning_in_a_bottle:potion_tracker/kill
```

Comment:
These unconditional 20Hz `@e`'s are still unfavorable. Though I don't see a way of making these conditional reliably.